readme.txt

```diff
@@ -8,37 +8,35 @@ Stable tag: STABLETAG
-* Deliver dynamically created responsive images across various devices, in different resolutions with exact pixel accuracy.
-* Lazy load images automatically with customizable placeholder settings.
-* Automatically optimize existing videos and images, as well as any future uploads.
-* Offer image and video editing capabilities, powered by AI, including transcoding, smart cropping, applying filters, generating thumbnails, and more.
+> * Automatically optimize and transform all your new and existing images and videos through best-in-class algorithms that improve site performance and SEO rankings. [Check out some examples](https://cloudinary.com/developers)!
+* Rapidly edit assets, via AI, that deepen engagement through capabilities such as smart cropping, thumbnail generation, automated transcoding, and more.
+* Deliver dynamically created responsive images across various devices and different resolutions.
+* Utilize lazy loading and customizable placeholders to load images that are viewed by your users.
+* Support both [headless](https://cloudinary.com/blog/image-optimization-headless-wordpress-wpgraphql) and UI-based WordPress deployments, maintaining flexibility in website development.
 * Maintain compatibility with popular page builders.
 * Provide end-to-end support for the WooCommerce plugin.
+* Stream videos using the Cloudinary Video Player, featuring advanced capabilities to enhance user engagement.
+* Present a responsive and interactive Product Gallery for creating captivating visual experiences.
 
 == About ==
 
-Cloudinary’s [award-winning plugin](https://www.businesswire.com/news/home/20200923005566/en/Cloudinary-Wins-2020-MarTech-Breakthrough-Award-for-Best-WordPress-Plugin)  makes automating your image, video, and visual media workflows easy, offering a variety of features. From image and video upload through management, transformation, optimization, and global CDN delivery, all features are offered within the plugin. Users can get started in minutes with a true plug-and-play solution paired with an intuitive onboarding wizard helping deliver optimized and responsive media out-of-the-box.
+Cloudinary’s [award-winning plugin](https://www.businesswire.com/news/home/20200923005566/en/Cloudinary-Wins-2020-MarTech-Breakthrough-Award-for-Best-WordPress-Plugin) makes automating your image and video workflows easy, offering a variety of features. Supporting nearly two million developers and over 10,000 customers, Cloudinary enables companies to manage, transform, optimize and deliver image and video assets. The Cloudinary Plugin supports all these capabilities, providing users with a true plug-and-play solution. Getting started takes only minutes using the intuitive onboarding wizard.
 
 https://youtu.be/AMVS_O_VEss
 
 > Know more on at the [Cloudinary Academy - Introduction to Cloudinary for WordPress Administrators](https://training.cloudinary.com/courses/introduction-to-cloudinary-for-wordpress-administrators-70-minute-course-zf3x)
```

Comment:
Maybe replace "Know more on at the [Cloudinary Academy..." with "Learn more about the plugin with the [Cloudinary Academy..."